runtime/vm/criuhelpers.cpp

```diff
@@ -137,7 +136,7 @@ addInternalJVMCheckpointHook(J9VMThread *currentThread, BOOLEAN isRestore, J9Cla
 		newHook->instanceType = instanceType;
 		newHook->includeSubClass = includeSubClass;
 		newHook->hookFunc = hookFunc;
-		/* newHook->instanceObjects is to be lazy initialized */
+		/* newHook->instanceObjects is to be lazily initialized */
```

Comment:
For this PR consistence, please add a period here as well.

Reply:
That comment doesn't begin with a capital, so it's not a sentence that needs to end with a period. There are several examples like that, the first of which is on line 57.